labellab-server/.gitignore

```diff
@@ -246,3 +246,6 @@ $RECYCLE.BIN/
 
 # User
 config/keys.js
+
+# Asset uploads
+public/uploads/
```

Comment:
@2knal We can't ignore this directory from the source control since the image upload functionality depends on this and have an effect on current continuous delivery configs.  So it's better to keep it as it is.

Reply:
@UdeshUK, can you please review the recent changes?